### ISSUE TITLE
[Question] When finetuning with my custom dataset, do I need to mix the custom datasets and original LLaVA instruction datasets?

### ISSUE THREAD

**Issue by zy1996829** [haotian-liu/LLaVA#1658]
### Question

When I only used the customized datasets, overfitting was very serious. Even if I set the epochs to be very small, the final loss was controlled at around 0.6, and overfitting was still very serious. @haotian-liu
